Unit Tests/Tests/TestDECHashSHA3.pas

```diff
@@ -24,6 +24,7 @@ interface
 {$INCLUDE TestDefines.inc}
 {$INCLUDE ..\..\Source\DECOptions.inc}
 
+{$IF CompilerVersion >= 24.0} // Too many local constants for Delphi XE2 and older
```

Comment:
Can you please explain this comment?
Which are those constants you refer to? Are some of 
the string literals assigned in the code too long?
Would splitting them up be an alternative solution?
If so, where's the limit?

Reply:
`[DCC Error] TestDECHashSHA3.pas(43924): E2283 Too many local constants.  Use shorter procedures` in `TestTHash_SHA3_224.SetUp`.

It's not that the string literals are too long, there are simply too many literals.

From the documentation:
> The DCC32 and DCCOSX compilers produce E2283 when the code generator gives up after literal/const values exceed 64Kbytes. This 64K limitation applies to string, set, and floating-point values in expressions of a function body, and the limitation is per function.

This is from the Sydney docs; does the Delphi 10.4 32-bit compiler not raise this error?

In my opinion it would make more sense to load and parse the rsp files at runtime rather than hardcoding the vectors.